WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -314,17 +314,18 @@ def payment_setup
 
     # Paypal setup URL
     # TODO: Don't generate this URL if there is already a connected paypal account?
-    @paypal_onboarding_url = PaypalInterface.generate_paypal_onboarding_link(@competition.id) unless Rails.env.production?
+    @paypal_onboarding_url = PaypalInterface.generate_paypal_onboarding_link(@competition.id) unless EnvConfig.WCA_LIVE_SITE?
```

Comment:
If your goal is to allwo tests in staging, you should perform _both_ checks to be sure. Something along the lines of `unless Rails.env.production? && EnvConfig.WCA_LIVE_SITE?`

Reply:
Changed to `paypal_enabled?`